example-app/src/main/java/com/adyen/checkout/example/data/api/model/AdditionalData.kt

```diff
@@ -8,6 +8,9 @@
 
 package com.adyen.checkout.example.data.api.model
 
+import androidx.annotation.Keep
+
+@Keep
```

Comment:
Is this better than keeping all classes in `com.adyen.checkout.example.data.api.model`?

Reply:
It's just preference. I like this more because it's explicit and if you move classes you don't have to manually update the proguard rules. For `LogoSize` I did add a proguard rule though, because I am not sure if `@Keep` works well in libraries